### DISCUSSION TITLE
GraphQL mutation support

### DISCUSSION THREAD

**Discussion by TomDeSmet** [solspace/craft-calendar#98] [Feature Requests]
Are there any plans to integrate GraphQL mutations for events?
E.g. we have events that have a users field where users are stored that will participate in that event.
We cannot use GraphQL now to mutate an event and add those users.

**Comment by kjmartens** [comment]
Hi @TomDeSmet,

This is something on our list for future consideration. With Craft 4 on the horizon, it might be something we tackle then. 🙂

**Reply by NicoDG** [reply]
@kjmartens Any update on this topic?

**Comment by smunusamy** [comment]
Hi @kjmartens,
Any update on the mutation support with Craft 4, we are looking for this feature as well.

**Reply by kjmartens** [reply]
Sorry for the delay @smunusamy. We don't have any update on this yet, unfortunately.

**Reply by smunusamy** [reply]
Following up for any new updates on this item

**Reply by kjmartens** [reply]
Not yet, unfortunately.
